{"name": "test_login_error2[data0]", "status": "passed", "steps": [{"name": "获取返回的登录页面", "status": "passed", "start": 1570275615014, "stop": 1570275615014}, {"name": "输入错误的用户名与密码，点击登录", "status": "passed", "start": 1570275615014, "stop": 1570275615340}, {"name": "获取登录失败后错误的提示", "status": "passed", "start": 1570275615340, "stop": 1570275615856}, {"name": "断言", "status": "passed", "start": 1570275615856, "stop": 1570275615868}], "parameters": [{"name": "data", "value": "{'user': '13423456789', 'pwd': '123', 'expected': '此账号没有经过授权，请联系管理员!'}"}], "start": 1570275615014, "stop": 1570275615868, "uuid": "72f5c46f-832e-4b92-9103-e36a1b11a47e", "historyId": "a21a98fa503f63b7519add614430e1ff", "testCaseId": "6609d051087a1db17f7a1e5cac185aee", "fullName": "test_cases.test_login.TestLogin#test_login_error2", "labels": [{"name": "story", "value": "登录失败用例"}, {"name": "feature", "value": "测试登录功能"}, {"name": "tag", "value": "success"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-I3J3MGM"}, {"name": "thread", "value": "2924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_login"}]}